Formula/basisu.rb

```diff
@@ -0,0 +1,20 @@
+class Basisu < Formula
+  desc "Basis Universal GPU texture codec command-line compression tool"
+  homepage "https://github.com/BinomialLLC/basis_universal"
+  url "https://github.com/BinomialLLC/basis_universal/archive/refs/tags/v1.15_rel2.tar.gz"
+  version "1.15-rel2"
```

Comment:
Is `-rel2` really part of the version? The [release](https://github.com/BinomialLLC/basis_universal/releases/tag/v1.15_rel2) just says `v1.15`.

Reply:
If you look at the previous releases, there have been a few with the same version number but different release strings on the end, so I was playing it safe. I don't mind to be honest.

Reply:
Changed version number to 1.15 since there were no other releases containing this particular version number